### ISSUE TITLE
Redesign the disease prediction page 

### ISSUE THREAD

**Issue by mohit200008** [mohit200008/medi-Care#45]
The cards for predictions have to be positioned and they can be in form of square cards.

**Comment by simran2712** [comment]
Hii, I request you to please assign this issue to me..

**Comment by mohit200008** [comment]
Sure @simran2712  go ahead

**Comment by mohit200008** [comment]
Any update on this @simran2712

**Comment by simran2712** [comment]
Sir, I'm unable to understand what exactly has to be modified. If possible, tell me about the modifications, I'll definitely do it

**Comment by mohit200008** [comment]
You can make use of the white space and put a carousel with pictures related to diseases and medical. And below that the cards can be present in horizontal alignment and to be designed and animated such that it looks more appealing.

**Comment by nawed2611** [comment]
Hi! Is this issue still open? I wanted to work on it.

**Comment by mohit200008** [comment]
YES YOU CAN WORK ON IT @nawed2611  THE ISSUE IS OPEN BUT KWOC IS OVER NOW

**Comment by nawed2611** [comment]
On it!

**Comment by nawed2611** [comment]
Kindly review #48

**Comment by sulogna2001** [comment]
Hey Can you assign me this issue for GSsoc22

**Comment by Ghanshyam88** [comment]
Hello 👋
Is this issue still open.
If it is I would like to work on this issue.

**Comment by BhumikaJena** [comment]
Hey, I would like to make some improvements in  the disease prediction  page and make it more appealing for the users . I would be glad if u assign this project to me.
I am a GSSOC'22 participant.

**Comment by mohit200008** [comment]
go ahead @BhumikaJena  and share a demo before making a pr of the page

**Comment by BhumikaJena** [comment]
Thank you for assigning me.

Sent from Mail for Windows

**Comment by Diishasing** [comment]
hi @mohit200008 
Please let me work on this feature. I am a GSSOC'22 participant. I can really learn a lot from this work.

**Comment by mohit200008** [comment]
Yes go ahead @likhithaindukuri

**Comment by BhuvanAde** [comment]
I believe I could work on this. Let me know If I can work on this issue.

**Comment by mohit200008** [comment]
Yes go ahead
